cli/command/client/map.go

```diff
@@ -106,11 +108,25 @@ func ParseSize(size string) (int, error) {
 	return n, nil
 }
 
+func ParseBlockSize(blocksize string) (uint64, error) {
+	blocksize = strings.TrimSuffix(blocksize, "B")
+	m, err := humanize.ParseBytes(blocksize)
+	if err != nil || m <= 0 {
+		return 0, errno.ERR_VOLUME_BLOCKSIZE_REQUIRES_POSITIVE_INTEGER.
+			F("blocksize: %s", humanize.Bytes(m))
```

Comment:
humanize.IBytes(m)

Reply:
Learn the difference between Bytes and IBytes:[Multiple-byte units](https://en.wikipedia.org/wiki/Byte#Multiple-byte_units)